rpcd/playbooks/roles/rpc_maas/templates/filebeat_process_check.yaml.j2

```diff
@@ -0,0 +1,19 @@
+type: agent.plugin
+label: "filebeat_process_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : process_check.py
+    args    : [ {% for name in filebeat_process_names %}{{ name }}{% if not loop.last %}, {% endif %}{% endfor %}, {%if "{{ ansible_ssh_host }}" in groups["all_containers"]%} -c "{{ ansible_ssh_host }}" {% else %}{% endif %}]
```

Comment:
NIT: Right now, inside the filebeat check files, this line prints args : [rsyslogd, ].
If you put the comma inside that proceeding conditional, you won't have the trailing comma.